cmd/migrate_to_wallets.go

```diff
@@ -0,0 +1,112 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	ljsonrpc "github.com/lbryio/lbry.go/v2/extras/jsonrpc"
+	"github.com/lbryio/lbrytv/internal/lbrynet"
+	"github.com/lbryio/lbrytv/models"
+
+	"github.com/spf13/cobra"
+	"github.com/volatiletech/sqlboiler/boil"
+)
+
+func init() {
+	rootCmd.AddCommand(migrateToWallets)
+}
+
+func prettyPrint(i interface{}) {
+	s, _ := json.MarshalIndent(i, "", "\t")
+	fmt.Println(string(s))
+}
+
+func ptrToBool(b bool) *bool { return &b }
+
+var migrateToWallets = &cobra.Command{
+	Use:    "migrate_to_wallets",
+	Short:  "Migrate existing accounts to wallets",
+	Hidden: true,
+	Run: func(cmd *cobra.Command, args []string) {
+		realRun := false
+		if len(args) > 0 && args[0] == "doit" {
+			realRun = true
+		}
+		c := lbrynet.Client
+
+		usersQ := models.Users(models.UserWhere.WalletID.EQ(""))
+		users, err := usersQ.AllG()
```

Comment:
i usually do this as one line: `users, err := models.Users(models.UserWhere.WalletID.EQ("")).AllG()`